packages/odyssey-storybook/src/contributing/LabsComponents.stories.mdx

```diff
@@ -0,0 +1,40 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Contributing/Labs Components" />
+
+# Contibuting Odyssey Labs Components
+
+If you are wanting to add components to Odyssey Labs, read this doc first before creating a PR.
+
+## Requirements
+
+Getting components added to Odyssey requires a few steps to be completed first:
+
+1. Get approval from Odyssey design to see if that component even belongs in Odyssey.
```

Comment:
Suggest defining the purpose of `Odyssey Labs`. It is unclear to me what is and is not appropriate for `Odyssey Labs`. 

For instance, is an AutoComplete providing a rich `Find apps` experience using Okta public APIs appropriate for `Odyssey Labs`?

If we're not sure yet, suggest erring toward `anything being built for Okta products on public APIs is appropriate for Odyssey Labs` - then introducing a second level of granularity later, like `Odyssey-Beta`, i.e. things that were found useful and relevant for base design system.

I know we've gone back and forth a bunch here.

cc: @rchild-okta

Reply:
0bc30f4e6156b1015028083504b56a3bc2c5f003